Code/GeorgiaLibrarySystem-/GTLService/DataManagement/Code/LendingDm_Code.cs

```diff
@@ -50,5 +51,45 @@ public bool ReturnBook(int copyId)
                 return false;
             }
         }
+
+        public bool NoticeFilling()
+        {
+            foreach (var borrow in _lendingDa.GetAllActiveBorrows())
+            {
+                Member member = _memberDa.GetMember(borrow.SSN);
+                
+                if (DateTime.Now >= borrow.FromDate.AddDays(member.MemberType.LendingLenght + member.MemberType.GracePeriod) && borrow.noticeSent == null)
+                {
+                    borrow.noticeSent = false;
+                }
+            }
+            return _lendingDa.SaveBorrowChanges();
+        }
+    }
+
+    public class NoticeTimer
```

Comment:
I know its just a helper class, but i think it should be in model layer, not in dm

Reply:
but model layer cant call service

Reply:
then perhaps you should add a timerDm as well

Reply:
ok that i can do

Reply:
enything else i should fix/do in this pr?